TweenController/TweenController.swift

```diff
@@ -85,17 +85,17 @@ public class TweenController {
             }
         }
         
-        boundaries.forEach { $0.block(progress: progress) }
+        boundaries.forEach { $0.block(progress) }
     }
 }
 
 extension TweenController: DescriptorRegistration {
     
-    func register<T : Tweenable>(descriptor: TweenDescriptor<T>) {
+    func register<T : Tweenable>(_ descriptor: TweenDescriptor<T>) {
         descriptors.append(descriptor)
     }
     
-    func observe(boundary: Boundary) {
+    func observe(_ boundary: Boundary) {
```

Comment:
I think we should not remove the first argument labels for any of the functions above.